packages/snap-preact-components/src/components/Organisms/MobileSidebar/MobileSidebar.stories.tsx

```diff
@@ -0,0 +1,227 @@
+			table: {
+				type: {
+					summary: 'string',
+				},
+				defaultValue: { summary: '650px' },
+			},
+			control: { type: 'text' },
+		},
+		hideTitle: {
+			defaultValue: false,
+			description: 'hides the sidebar title component',
+			table: {
+				type: {
+					summary: 'boolean',
+				},
+				defaultValue: { summary: false },
+			},
+			control: { type: 'boolean' },
+		},
+		hideApplyButton: {
+			defaultValue: false,
+			description: 'hides the apply facets button component',
+			table: {
+				type: {
+					summary: 'boolean',
+				},
+				defaultValue: { summary: false },
+			},
+			control: { type: 'boolean' },
+		},
+		hideClearAllButton: {
+			defaultValue: false,
+			description: 'hides the clear all filters button component',
+			table: {
+				type: {
+					summary: 'boolean',
+				},
+				defaultValue: { summary: false },
+			},
+			control: { type: 'boolean' },
+		},
+		hideCloseButton: {
+			defaultValue: false,
+			description: 'hides the close sidebar icon component',
+			table: {
+				type: {
+					summary: 'boolean',
+				},
+				defaultValue: { summary: false },
+			},
+			control: { type: 'boolean' },
+		},
+		hideFacets: {
+			defaultValue: false,
+			description: 'hides the sidebar facets component',
+			table: {
+				type: {
+					summary: 'boolean',
+				},
+				defaultValue: { summary: false },
+			},
+			control: { type: 'boolean' },
+		},
+		hidePerPage: {
+			defaultValue: false,
+			description: 'hides the sidebar per page component',
+			table: {
+				type: {
+					summary: 'boolean',
+				},
+				defaultValue: { summary: false },
+			},
+			control: { type: 'boolean' },
+		},
+		hideSortBy: {
+			defaultValue: false,
+			description: 'hides the sidebar sort by component',
+			table: {
+				type: {
+					summary: 'boolean',
+				},
+				defaultValue: { summary: false },
+			},
+			control: { type: 'boolean' },
+		},
+		hideFilterSummary: {
+			defaultValue: false,
+			description: 'hides the sidebar FilterSummary component',
+			table: {
+				type: {
+					summary: 'boolean',
+				},
+				defaultValue: { summary: false },
+			},
+			control: { type: 'boolean' },
+		},
+		iconOpen: {
+			description: 'Icon name for the open side bar icon, renders in the slideout button.',
+			table: {
+				type: {
+					summary: 'string',
+				},
+			},
+			control: {
+				type: 'select',
+				options: [...Object.keys(iconPaths)],
+			},
+		},
+		iconClose: {
+			description: 'Icon name for the close side bar icon, renders in the sidebar title.',
+			table: {
+				type: {
+					summary: 'string',
+				},
+			},
+			control: {
+				type: 'select',
+				options: [...Object.keys(iconPaths)],
+			},
+		},
+		...componentArgs,
+	},
+};
+
+const snapInstance = Snapify.search({ id: 'Facet', globals: { siteId: '8uyt2m' } });
+
+export const Default = (args: MobileSidebarProps, { loaded: { controller } }: { loaded: { controller: SearchController } }) => {
+	return <MobileSidebar {...args} controller={controller} />;
```

Comment:
For this default example in storybook, I think changing the openButtonText could be helpful; maybe something like 'click me' or 'click to open filters' or similar to make it more obvious.